From 79d85c51bfc497fc0b5f4a0f1e35c4d4fdcc835d Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Wed, 23 Nov 2011 14:21:48 +0800
Subject: [PATCH 0660/2463] ENGR00161617_1 [MX6] : Get FEC MAC addr from the
 shadow reg.

- Correct FEC to get the default MAC address from OCOTP
  shadow reg.

  Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 arch/arm/mach-mx6/mx6_fec.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/mx6_fec.c b/arch/arm/mach-mx6/mx6_fec.c
index 32e8583..b2aa45b 100644
--- a/arch/arm/mach-mx6/mx6_fec.c
+++ b/arch/arm/mach-mx6/mx6_fec.c
@@ -25,7 +25,7 @@
 #include <asm/mach/arch.h>
 #include "devices-imx6q.h"
 
-#define HW_OCOTP_MACn(n)        (0x00000250 + (n) * 0x10)
+#define HW_OCOTP_MACn(n)        (0x00000620 + (n) * 0x10)
 
 static int fec_get_mac_addr(unsigned char *mac)
 {
-- 
1.7.7.4

